src/pages/my_page/user_profile/MyProfile.tsx

```diff
@@ -1,46 +1,167 @@
+          gridTemplateAreas: {
+            xs: '"user-profile" "research-profile"',
+            md: '"user-profile research-profile" ',
+          },
+        }}>
         <BackgroundDiv
           sx={{
             display: 'grid',
-            columnGap: '3rem',
             gridTemplateAreas: {
               xs: '"primary-info" "roles"',
-              md: '"roles primary-info"',
+              md: '"roles" "primary-info"',
             },
-            gridTemplateColumns: { xs: '1fr', md: '1fr 2fr' },
+            bgcolor: 'info.light',
+            gridArea: 'user-profile',
           }}>
-          <Box sx={{ gridArea: 'roles' }}>
-            <UserRoles user={user} />
-          </Box>
-
-          <Box sx={{ display: 'grid', gridArea: 'primary-info', gap: '1rem' }}>
-            <UserInfo user={user} />
-            <Divider />
-            <UserOrcid user={user} />
+          <Box sx={{ display: 'flex', flexDirection: 'column', gap: '1.5rem' }}>
+            <Typography variant="h2">{t('my_page.my_profile.heading.personalia')}</Typography>
+            {isLoadingPerson && !person ? (
+              <PageSpinner />
+            ) : (
+              <>
+                <Typography>{t('my_page.my_profile.user_profile_description')}</Typography>
+                <Formik initialValues={initialValues} onSubmit={updatePerson} enableReinitialize>
+                  {({ isSubmitting, dirty }: FormikProps<CristinPersonFormData>) => (
+                    <Form>
+                      <Box sx={{ display: 'flex', gap: '1rem' }}>
+                        <Field name={'preferredFirstName'}>
+                          {({ field, meta: { touched, error } }: FieldProps<string>) => (
+                            <TextField
+                              {...field}
+                              id={field.name}
+                              disabled={!editPreferredNames || isSubmitting}
+                              label={t('my_page.my_profile.preferred_first_name')}
+                              size="small"
+                              variant="filled"
+                              error={touched && !!error}
+                              helperText={<ErrorMessage name={field.name} />}
+                            />
+                          )}
+                        </Field>
+                        <Field name={'preferredLastName'}>
+                          {({ field, meta: { touched, error } }: FieldProps<string>) => (
+                            <TextField
+                              {...field}
+                              id={field.name}
+                              disabled={!editPreferredNames || isSubmitting}
+                              label={t('my_page.my_profile.preferred_last_name')}
+                              size="small"
+                              variant="filled"
+                              error={touched && !!error}
+                              helperText={<ErrorMessage name={field.name} />}
```

Comment:
Siden vi ikke har noe validering her nå tenker jeg vi kan forenkle ved å bare droppe logikken for å vise feilmeldinger også
```suggestion
                          {({ field }: FieldProps<string>) => (
                            <TextField
                              {...field}
                              id={field.name}
                              disabled={!editPreferredNames || isSubmitting}
                              label={t('my_page.my_profile.preferred_first_name')}
                              size="small"
                              variant="filled"
                            />
                          )}
                        </Field>
                        <Field name={'preferredLastName'}>
                          {({ field }: FieldProps<string>) => (
                            <TextField
                              {...field}
                              id={field.name}
                              disabled={!editPreferredNames || isSubmitting}
                              label={t('my_page.my_profile.preferred_last_name')}
                              size="small"
                              variant="filled"
```